maud_macros/tests/warnings.rs

```diff
@@ -0,0 +1,15 @@
+extern crate compiletest_rs;
+
+use std::path::PathBuf;
+
+#[test]
+fn run_warnings() {
+    let mut config = compiletest_rs::Config::default();
+
+    config.mode = compiletest_rs::common::Mode::Ui;
+    config.src_base = PathBuf::from("tests/warnings");
+    config.link_deps(); // Populate config.target_rustcflags with dependencies on the path
+    config.clean_rmeta(); // If your tests import the parent crate, this helps with E0464
```

Comment:
The error with multiple feature crates still happens - but this might simplify the structure and the Travis file as well, since testing wouldn't be as out of order. Would you prefer it?

Reply:
Yeah that would be great 👍